src/stryker-cli.ts

```diff
@@ -25,40 +28,40 @@ export function run(log = console.log): Promise<void> {
 }
 
 function runLocalStryker() {
-  const strykerCli = localStrykerCli();
-  RequireWrapper.require(strykerCli);
+  const stryker = localStryker();
+  NodeWrapper.require(stryker);
   return Promise.resolve();
 }
 
-function localStrykerCli() {
-  const dirname = path.dirname(RequireWrapper.resolve('stryker'));
+function localStryker() {
+  const stryker = resolve.sync('stryker', { basedir: NodeWrapper.cwd() });
```

Comment:
Perhaps `resolve.sync` can be moved to NodeWrapper too.

Reply:
Well.. it's not strictly necessary. The other once needed to be moved, because they cannot be (easily) mocked away for testing. Also, if we were to move it, we would need to find a different name, because we would not be *only* wrapping Node ;)

Reply:
Sounds reasonable. Let's keep it how it is.